net/vallumd/files/vallumd.conf

```diff
@@ -0,0 +1,5 @@
+config vallumd mqtt
+	option host 127.0.0.1
+	option port 1883
+	list topics blacklist4
+	list topics blacklist6
```

Comment:
So you suggest to add optoin disabled '1'? Or add something like this to the Makefile:
```
define Package/vallumd/postinst
#!/bin/sh
# check if we are on real system
if [ -z "$${IPKG_INSTROOT}" ]; then
        echo "Disabling rc.d symlink for vallumd"
        /etc/init.d/vallumd disable
fi
exit 0
endef
```